<launch>
	<node pkg="tf" type="static_transform_publisher" name="openni_base_link" args="0.0 0.0 1.59 0 0.785 0 /world /openni_camera 100" />

	<node name="ar_kinect_bluetooth" pkg="nxt_localization" type="ar_kinect_bluetooth" respawn="false" output="screen">
		<param name="marker_pattern_list" type="string" value="$(find nxt_localization)/ros/cfg/markers.cfg"/>
		<param name="marker_data_directory" type="string" value="$(find ar_pose)/"/>
		<param name="threshold" type="int" value="100"/>
		<param name="publish_tf" type="int" value="1"/>
		<param name="publish_visual_markers" type="int" value="0"/>
		<param name="publish_ar_pose_makers" type="int" value="0"/>
		<param name="publish_to_bluetooth" type="int" value="1"/>
	</node>
</launch>
